.travis.yml

```diff
@@ -3,24 +3,31 @@ dist: xenial
 
 python:
   - "2.7"
+  - "3.8"
+
+services:
+  - mysql
 
 install:
-  - sudo pip install urllib3 pyOpenSSL ndg-httpsclient pyasn1
+  - pip install --upgrade pip setuptools
+  - pip install cffi cryptography flake8 ndg-httpsclient pyasn1 pycparser pyOpenSSL urllib3
   - sudo apt-get purge -y mysql-common mysql-server mysql-client
   - sudo apt-get install --only-upgrade -y git
   - sudo apt-get install hhvm && rm -rf /home/travis/.kiex/
   - mkdir -p ~/.bench
   - mkdir -p /tmp/.bench
+  - flake8 . --count --select=E9,F63,F7,F822,F823 --show-source --statistics
+  - # TODO: (cclauss) add F821 to the testing when issue #856 is fixed
+  - flake8 . --count --exit-zero --select=F821 --show-source --statistics
   - cp -r $TRAVIS_BUILD_DIR/* ~/.bench
   - cp -r $TRAVIS_BUILD_DIR/* /tmp/.bench
 
-  - sudo python $TRAVIS_BUILD_DIR/playbooks/install.py --user travis --run-travis --production --verbose
+  # - sudo python $TRAVIS_BUILD_DIR/playbooks/install.py --user travis --run-travis --production --verbose
```

Comment:
Given that we are in a virtualenv, we do not need sudo on pip.  As always, the use of sudo should be minimized.
https://docs.travis-ci.com/user/languages/python/#travis-ci-uses-isolated-virtualenvs

Reply:
You have removed the line.

Reply:
Sorry.  You are correct.  When the line is in place then __bench__ fails the build.  I am trying to understand how to avoid this because __bench__ seems to have troubles.

__Bench__'s CircleCI tests have been failing for the past 4 months.
https://circleci.com/gh/organizations/frappe or should the .circleci directory be removed?

__Bench__'s Travis CI tests stopped working 11 months ago.
https://travis-ci.org/frappe/bench/builds and probably needs to be migrated to travis-ci.com
https://travis-ci.com/frappe/bench or remove __.travis.yml__

My usage is dev/test/evaluation.